vllm/entrypoints/llm.py

```diff
@@ -233,22 +233,29 @@ def _run_engine(self, use_tqdm: bool) -> List[RequestOutput]:
         # Initialize tqdm.
         if use_tqdm:
             num_requests = self.llm_engine.get_num_unfinished_requests()
-            pbar = tqdm(total=num_requests,
-                        desc="Processed prompts",
-                        dynamic_ncols=True)
+            pbar = tqdm(
+                total=num_requests,
+                desc="Processed prompts",
+                dynamic_ncols=True,
+                postfix=f"{0} toks/s",
```

Comment:
Just to make the starting postfix match formatting
```suggestion
                postfix=f"{0:.2f} toks/s",
```